_posts/2017-12-28-foreman-community-newsletter---december-2017.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Foreman Community Newsletter - December 2017
+date: 2017-12-28 15:10:00
+author: Ori Rabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Migrating from mailing lists, Katello 3.5, and the usual round-up
+---
+
+Happy holidays!
+
+### Migrating from Google Groups
+
+Our mailing lists are moving to Discourse on 1st Jan!
+See the [announcement][discourse_announcement] for more details.
+
+For those who missed it, the past month there were discussions and
+demos showing Discourse. After a migration poll last month the move to
+Discourse got a lot of support.
+The results were discussed [here](https://youtu.be/g8XnAKfG9xY?t=99). 
+
+### Katello 3.5 released
+
+On the 7th of Dec Katello 3.5 was released, see the [post][k_3_5] for details.
+The release fixed 28 bugs found in RC2 so thank you to all that tested,
+commented and fixed! As always, we'd love your feedback, so do report
+any issues you find to the [Katello bug tracker][k_issues]!
+
+### Upcoming Events
+
+* Community Demo - 4th Jan  (https://www.youtube.com/watch?v=BVTiwSEMcyw)
```

Comment:
Shouldn't this also be under the Community Demos header below? Also maybe make the link from the text?

Reply:
No, it's always like this. The next community demo is under upcoming events. If there were any other deep dives coming up they would also be here.